### DISCUSSION TITLE
CORTX could be used as a data recovery tool?

### DISCUSSION THREAD

**Discussion by yanqingfu** [Seagate/cortx#431] [Ideas] [answered]
Based on the information of  [Democratizing Access to Mass Capacity Object Storage](https://www.youtube.com/watch?v=h3JXk9r8nNc&list=PLOLUar3XSz2M_w5OxQLNDBTpSrsGbjDWr&index=2) video, it seems that CORTX could be used as a data recovery tool. Is this true?

**Comment by johnbent** [comment, marked as answer]
Hey @yanqingfu .  Absolutely.  CORTX is designed to be a highly scalable system that masks drive failures by rebuilding lost data using distributed declustered parity.  You can learn more about this here:
1. https://github.com/Seagate/cortx-motr/blob/main/doc/motr-in-prose.md#failure-idea
2. https://github.com/Seagate/cortx-motr/blob/main/doc/PDF/HLD_of_SNS_Repair.pdf
3. [Democratizing Access to Mass Capacity Object Storage - YouTube](https://www.youtube.com/watch?v=h3JXk9r8nNc&list=PLOLUar3XSz2M_w5OxQLNDBTpSrsGbjDWr&index=2)

Let me know if you still have more questions.  Thanks for asking!
